pubspec.yaml

```diff
@@ -14,14 +14,14 @@ platforms:
   linux:
 
 environment:
-  sdk: ">=3.2.0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
I think some of these dependencies need to change - I can only get this working on web, not on Android, iOS or macOS.
I'm not sure exactly which are causing this issue, but camerawersome and just_audio seem to be unhappy for me

Reply:
updated pods for example